dev/packages/alpha/system/0.0.1/dataset/auth/agent/stream/log.yml.hbs

```diff
@@ -0,0 +1,14 @@
+paths:
+{{#each paths as |path i|}}
+ - {{path}}
+{{/each}}
+exclude_files: [".gz$"]
+multiline:
+  pattern: "^\\s"
+  match: after
+processors:
+  - add_locale: ~
+  - add_fields:
+      target: ''
+      fields:
```

Comment:
This is interesting. I wonder if we should add this still on the Beats side in the future. @ph